addons/test_discuss_full/tests/test_performance.py

```diff
@@ -1408,7 +1408,6 @@ def _expected_result_for_message(self, channel):
                 "starredPersonas": [{"id": self.users[0].partner_id.id, "type": "partner"}],
                 "subject": False,
                 "subtype_description": False,
-                "subtype_id": [mt_note_id, "Note"],
```

Comment:
`mt_note_id` and `mt_comment_id` variable are probably left unused now?

Reply:
Ho yes, totally, I m going to remove them